src/sage/groups/finitely_presented.py

```diff
@@ -1348,7 +1353,108 @@ def abelian_invariants(self):
         Uses GAP.
         """
         invariants = self.gap().AbelianInvariants()
-        return tuple( i.sage() for i in invariants )
+        return tuple(i.sage() for i in invariants)
+
```

Comment:
We follow one blankline between methods.

Reply:
I did not find the missing blankline

Reply:
When I looked at this before, there were two blanklines IIRC. In either case, it is moot now.